charts/redpanda/values.go

```diff
@@ -166,10 +249,22 @@ type ExternalConfig struct {
 type Logging struct {
 	LogLevel    string `json:"logLevel" jsonschema:"required,pattern=^(error|warn|info|debug|trace)$"`
 	UseageStats struct {
-		Enabled bool `json:"enabled" jsonschema:"required"`
+		Enabled      bool    `json:"enabled" jsonschema:"required"`
+		Organization *string `json:"organization"`
+		ClusterID    *string `json:"clusterId"`
 	} `json:"usageStats" jsonschema:"required"`
 }
 
+func (l *Logging) Translate() map[string]any {
+	result := map[string]any{}
+
+	if clusterID := ptr.Deref(l.UseageStats.ClusterID, ""); clusterID != "" {
```

Comment:
Should this also include organization?

Reply:
No, as the organization was removed from Redpanda